autoload/maktaba/extension.vim

```diff
@@ -0,0 +1,159 @@
+""
+" @dict ExtensionRegistry
+" A registry for the extensions used a single plugin.  Extensions are
+" dictionaries with a plugin-specific interpretation
+"
+" Plugins should use |Plugin.GetExtensionRegistry| to gain access to their own
```

Comment:
I want the dict method on Plugin, not maktaba#extensions#GetExtensionRegsitry() (which doesn't exist).  Is there a better way to refer to that?